egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -314,30 +314,49 @@ def train(args, run_opts, background_process_handler):
     default_egs_dir = '{0}/egs'.format(args.dir)
     if (args.stage <= -3) and args.egs_dir is None:
         logger.info("Generating egs")
+        # If num of egs dirs in args.egs_dir > 1,
+        # it is correspond to multilingual training and 
+        # it should generate egs dir for each languages.
+        # The last dir corresponds to multilingual egs directory,
+        # that is generated using this script, but 
+        # it requires single egs dirs to exist.
+        multi_egs_dir = args.egs_dir.split()
```

Comment:
I don't really like this script having to deal with the multilingual egs.  I was thinking of moving the egs-generation out of the train.py scripts anyway, as it's kind of unrelated to the rest of what they do.
Can't we just insist that in the multilingual case, the egs must already have been generated,  and the dir must be supplied using the common-egs-dir option?